src/components/App/App.js

```diff
@@ -2,42 +2,58 @@ import React, {Component} from 'react';
 import './App.css';
 import Form from '../Form/Form';
 import Movies from '../Movies/Movies';
-import movieData from '../../data/movieData';
 import SingleMovie from '../SingleMovie/SingleMovie';
+import getData from '../../apiCalls/api';
 
 class App extends Component {
  constructor() {
   super();
   this.state = {
     allMovies: [],
     singleMovieId: 0,
-    singleMovie: {},
+    singleMovie: "",
```

Comment:
Is the single movie going to be a string when it's populated? ideally, the initial state's data type would match the data type it will always be.